parsers/html.c

```diff
@@ -20,6 +20,8 @@
 #include "keyword.h"
 #include "promise.h"
 
+#define MAX_DEPTH 1000
```

Comment:
Can I ask you to use a bit more long and descriptive name (or putting comments about the constant)?